### ISSUE TITLE
Links in issues don't work

### ISSUE THREAD

**Issue by rtzoeller** [welnanick/Issue-Manager-for-Github#19]
GitHub supports embedding and linking to a wide variety of media, including auto-linking commit hashes and also embedding photos.

Where possible, links should be displayed in issue comments.

**Comment by welnanick** [comment]
Yep. A lot of it will require me to parse the body of the comment to do proper linkage. I can see if I can get web links to work though.

**Comment by welnanick** [comment]
Partially addressed in 583d2e5, web links work now.
